src/main/java/edu/greenblitz/tobyDetermined/commands/rotatingBelly/RotateOutOfDoor.java

```diff
@@ -5,11 +5,23 @@
 import edu.wpi.first.wpilibj2.command.WaitCommand;
 
 public class RotateOutOfDoor extends ParallelDeadlineGroup {
+    private static final double endTimeSeconds = 0.5;
```

Comment:
broder - make deadlinegroupommands great again

Reply:
it's better like it is right now

Reply:
We need consistency, some of our code is using deadlinecommand, and this is not, I think the deadline command is more readable and easier to understand, but if you think otherwise please explain - just say "its better like this" is not doing that and is very unproductive. + I have no idea who wrote this comment but the last time I spoke with Tal he agreed with me.